Sources/MongoSwift/MongoDatabase.swift

```diff
@@ -77,18 +77,19 @@ public struct CreateCollectionOptions: Encodable, CodingStrategyProvider {
     /// to be inserted.
     public let validationAction: String?
 
-    /// Allows users to specify a default configuration for indexes when creating a collection.
+    /// Specify a default configuration for indexes created on this collection.
     public let indexOptionDefaults: Document?
 
     /// The name of the source collection or view from which to create the view.
     public let viewOn: String?
 
+    /// An array consisting of aggregation pipeline stages. When used with `viewOn`, will create the view by applying
+    /// this pipeline to the source collection or view.
+    public let pipeline: [Document]?
+
     /// Specifies the default collation for the collection.
     public let collation: Document?
 
-    /// A session to associate with this operation.
-    public let session: ClientSession?
```

Comment:
went ahead and removed this while I was here, given the new API. @patrickfreed, there are likely more of these cases in the existing API that we will need to remove.

Reply:
Yeah, I've removed them all in my wip sessions implementation